packages/flutter_tools/test/general.shard/runner/flutter_command_runner_test.dart

```diff
@@ -86,6 +88,40 @@ void main() {
         Platform: () => platform,
       }, initializeFlutterRoot: false);
 
+      testUsingContext('does not check that Flutter installation is up-to-date with CI=true in environment', () async {
+        final MockFlutterVersion version = globals.flutterVersion as MockFlutterVersion;
```

Comment:
@jonahwilliams Is there an alternative to `MockFlutterVersion`, yet.

Reply:
We don't have a good alternative. I think using the existing mock is fine until I write a replacement, since It should be straightforward for me to replace them in one go